server/@types/express/index.d.ts

```diff
@@ -59,12 +60,30 @@ export type ContributoryFactor = {
   comment?: string
 }
 
-type SaferCustodyScreening = object
+type SaferCustodyScreening = Partial<{
+  outcomeType: ReferenceData
+  reasonForDecision: string
+}>
 
 type Investigation = object
 
 type Plan = object
 
+export type CsipRecord = Partial<{
```

Comment:
There's a non obvious difference between the hand coded types in this file, which are session data internal to this UI code, and an API response that you're storing here. In this case you can add `export type CsipRecord = components['schemas']['CsipRecord']` to csipApiTypes.ts to use the types autogenerated via the OpenAPI docs then reference `CsipRecord` in `export type JourneyData`, probably after prisoner.

Prisoner is a special case because we only use one endpoint of the Prisoner Search API and only a tiny subset of the Prisoner response. That's why that one is hand coded rather than autogenerated